SPECS/catatonit/catatonit.spec

```diff
@@ -0,0 +1,91 @@
+Summary:        A signal-forwarding process manager for containers
+Name:           catatonit
+Version:        0.1.7
+Release:        7%{?dist}
+License:        GPLv3+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/openSUSE/catatonit
+Source0:        %{url}/archive/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  file
+BuildRequires:  gcc
+BuildRequires:  git
+BuildRequires:  glibc-static >= 2.35-3%{?dist}
+BuildRequires:  libtool
+BuildRequires:  make
+Provides:       podman-%{name} = %{version}-%{release}
+
+%description
+Catatonit is a /sbin/init program for use within containers. It
+forwards (almost) all signals to the spawned child, tears down
+the container when the spawned child exits, and otherwise
+cleans up other exited processes (zombies).
+
+This is a reimplementation of other container init programs (such as
+"tini" or "dumb-init"), but uses modern Linux facilities (such as
+signalfd(2)) and has no additional features.
+
+%prep
+%autosetup -Sgit
+sed -i '$d' configure.ac
+
+%build
+autoreconf -fi
+%configure
+#%{__make} %{?_smp_mflags}
```

Comment:
```suggestion
```

Reply:
Spec-cleaner suggested to replace with %make_build which I committed.  Are we sure to remove this?

Reply:
I have retained this else build fails.